### ISSUE TITLE
Typeshed branch import error

### ISSUE THREAD

**Issue by martinRenou** [davidhalter/jedi#1302]
I've got the following error when trying to import jedi (using the typeshed branch):
```python
~/github/martinRenou/jedi/jedi/_compatibility.py in <module>
     17 from zipimport import zipimporter
     18 
---> 19 from parso.file_io import KnownContentFileIO
     20 
     21 is_py3 = sys.version_info[0] >= 3

ModuleNotFoundError: No module named 'parso.file_io'
```
I could not find anything about `KnownContentFileIO` in the parso github repo history. Am I missing something? I'm using `parso=0.3.4` according the the requirement file.

**Comment by davidhalter** [comment]
Thanks for looking at this. My bad. I just pushed this to parso. I intend to release parso 0.3.5 very soon.

**Comment by martinRenou** [comment]
Thanks

**Comment by davidhalter** [comment]
Released parso 0.4.0.
